src/smriprep/workflows/surfaces.py

```diff
@@ -835,6 +839,7 @@ def init_msm_sulc_wf(*, sloppy: bool = False, name: str = 'msm_sulc_wf'):
     return workflow
 
 
+@tag('anat.gifti-surfaces')
 def init_gifti_surfaces_wf(
     *,
     surfaces: list[str] = ('pial', 'midthickness', 'inflated', 'white'),
```

Comment:
This workflow can be instantiated multiple times in a subject's workflow, e.g., with anatomical surfaces and with spheres. Tagging this here as opposed to after calling would mean that attempting to splice would replace two workflows with one.